### DISCUSSION TITLE
Layer norm not working in densenet

### DISCUSSION THREAD

**Discussion by MiguelMonteiro** [Project-MONAI/MONAI#8035] [General]
**Describe the bug**
Layer norm does not work with densenet, returns the following error when instantiating model:
```
TypeError: LayerNorm.__init__() missing 1 required positional argument: 'normalized_shape'
```

**To Reproduce**
```
from monai.networks.nets.densenet import densenet121
model = densenet121(spatial_dims=3, in_channels=1, out_channels=2048, norm="layer")
```
**Expected behavior**
It should work or not be an option.

**Environment**

Python 3.10, pytorch 2.0.1, monai 1.3.0

**Additional context**
Bug has previously been reported here https://github.com/Project-MONAI/research-contributions/issues/196

**Comment by KumoLiu** [comment]
Hi @MiguelMonteiro, @K-Rilla, I re-checked the issue and found that norm can accept a tuple as input that contains the parameters needed by the matching norm layer.
You can do something like this:
```
from monai.networks.nets.densenet import DenseNet121
model = DenseNet121(spatial_dims=3, in_channels=1, out_channels=2048, norm=("layer", {"normalized_shape": [c, h, d]}))
```

Convert it to discussion, feel free to create another issue if meet further issue. Thanks.
